eth/backend.go

```diff
@@ -149,7 +149,11 @@ func New(ctx *node.ServiceContext, config *Config) (*Ethereum, error) {
 		rawdb.WriteDatabaseVersion(chainDb, core.BlockChainVersion)
 	}
 	var (
-		vmConfig    = vm.Config{EnablePreimageRecording: config.EnablePreimageRecording}
+		vmConfig = vm.Config{
```

Comment:
Sorry, but you missed 64 other places where `vm.Config{}`. I'm not even sure if the new incoming transactions are going to be executed with the selected VM.

Reply:
@chfast those other location correspond to tests  and simulations and we want to use the native interpreter in them. Incoming transactions will be executed by the VM, if you don't believe us check it for yourself.

Reply:
I believe you. I just remember I had to change this in two places. Maybe it was about tests.